.github/workflows/docker.yml

```diff
@@ -210,12 +252,24 @@ jobs:
           type=ref,event=tag
           # short commit hash
           type=sha
-
-    - name: Build and push image
+    # The TR image is huge.
+    # We need to free up every last bit of space we can.
+    - name: Free disk space
+      uses: jlumbroso/free-disk-space@v1.2.0
+      with:
+        android: true
+        dotnet: true
+        haskell: true
+        large-packages: true
+        docker-images: true
+        swap-storage: true
+        tool-cache: false
+    - name: Build and push image (TR)
       # see https://github.com/docker/build-push-action
       uses: docker/build-push-action@v4
       with:
-        context: ./skema/text_reading/scala
+        context: ./skema/text_reading/scala/webapp/target/docker/stage
```

Comment:
I *think* that docker.sbt results in it being at /skema/webapp/app/bin/webapp.  I notice that right there I'm using a different dockerBaseImage.